src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -198,9 +199,15 @@ const fileUploadValidator = celebrate({
  */
 export const handleListDashboardForms: ControllerHandler<
   unknown,
-  AdminDashboardFormMetaDto[] | ErrorDto
+  AdminDashboardFormMetaDto[] | ErrorDto,
+  ApiReqBody
 > = async (req, res) => {
-  const authedUserId = (req.session as AuthedSessionData).user._id
+  let authedUserId: string
+  if (req.body.formSg?.userId) {
+    authedUserId = req.body.formSg?.userId
```

Comment:
The namespace `req.body` looks wrong 🤔 , can we shove the `formsg` namespace straight in `req`?

Reply:
Hmmm, in fact, instead of having to change this method to retrieve the user from 2 different objects, I'm thinking we could let the API auth put the authenticated user in the session object itself, so the old code would just work as-is.

Reply:
Thanks Tim, good point! I've put it in the session object itself - [32c1cc1](https://github.com/opengovsg/FormSG/pull/6376/commits/32c1cc1935b89e34e5abe9fa3803ea6520cef0e7)